### DISCUSSION TITLE
Create a tutorial for defense against the soldiers

### DISCUSSION THREAD

**Discussion by keyvankhademi** [WeAreMahsaAmini/Core#16] [Ideas]
1. Create a database of the videos shared on social media
2. ....

**Comment by farnoodma** [comment]
We can even ask experts to contribute (One of them is famous on Twitter).

**Comment by firiz** [comment]
Basic tools and hardware tutorial

**Comment by garshairv** [comment]
On top of that, we could also categorize different cities and populate videos of each city that people have been sharing on social media. This could potentially be represented as an informative website or a simple android/ios app.

**Comment by siamakhaz** [comment]
there are tons of youtube in the self-defence category we can just upload them to aparat.com

**Reply by behnazkhoshnood** [reply]
I find this link that might be useful here:
https://www.instagram.com/reel/CjG2QWJg-Ty/?utm_source=ig_web_copy_link

**Comment by ptkx132** [comment]
I think we should start by translating existing videos and make them easy to access for everyone 
we can create tutorial to educate protestors to:
1. get armed with basic tools
2. defend themselves 
3. how to survive
